pkgs/top-level/python-packages.nix

```
@@ -5328,6 +5328,8 @@ in {
 
   uarray = callPackage ../development/python-modules/uarray { };
 
+  ueberzug = callPackage ../development/python-modules/ueberzug { };
```

to avoid a python package of the same name colliding with your package
```suggestion
  ueberzug = callPackage ../development/python-modules/ueberzug {
    inherit (pkgs.xorg) libX11 libXext;
  };
```